pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -863,7 +935,7 @@ var _ = Describe("Transform", func() {
 			})
 		})
 
-		When("project service extension instructing to expose the k8s service with domain and annotations", func() {
+		When("project service extension instructing to expose the k8s service with domain and podAnnotations", func() {
```

Comment:
I suppose here we're dealing with IngressAnnotations so the context description is wrong here and below.

Reply:
That was a find and replace gone bad. Will fix.